### PR TITLE
David update his caption

### LINKED ISSUE #3: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @cody-carlson
- [ ] @cody-carlson-learning
- [ ] @ACaspeer
- [ ] @AndreiTeradyne
- [ ] @Daduc38
- [ ] @ElMT2023
- [ ] @JGT90
- [ ] @JameHann
- [ ] @JoelUsername
- [ ] @Raccoon137
- [ ] @frehiwotberhe
- [ ] @jgtres
- [ ] @joel8800
- [ ] @kbushe
- [ ] @mccroryn
- [ ] @rbelleau88
- [x] @shtaudor

### PR DESCRIPTION
**Pull request by Daduc38**
Create PR to fix Issue
Resolve https://github.com/teradyne-dev-workshops/caption-this-0109/issues/3
##Done

-Fix the code
-Commit the code

## To Do
[ ] Merge with Main Branch
[ ] Remove Branch

### CODE REVIEW

**Comment by Daduc38** [comment]
Need to reassign to Mehdi

**Comment by Daduc38** [comment]
Create PR to fix Issue
Resolve https://github.com/teradyne-dev-workshops/caption-this-0109/issues/3
##Done

##To Do
[  ] Merge with Main Branch
[  ] Remove Branch
-Fix the code
-Commit the code

To Do
[ ] Merge with Main Branch
[ ] Remove Branch

**Comment by Daduc38** [comment]
to close

**Comment by Daduc38** [comment]
Create PR to fix Issue
Resolve #3 
##Done
-Fix the code
-Commit the code
-PR Created
-Reassign to Correct Owner

##To Do
[ ] Merge with Main Branch
[ ] Remove Branch

File: _slides/06-Daduc38.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is my Great Capion !!!
```
Comment (ElMT2023) [inline _slides/06-Daduc38.md:6]: great Job

**Review by ElMT2023 — APPROVED** [review]

File: _slides/06-Daduc38.md (hunk 1 above)
Comment (Daduc38) [reply _slides/06-Daduc38.md:6]: - [ ] Thanks 😃

**Pull request closed without merging**